docs/community/tutorials-contest/2022/local-subnet-development/README.md

```diff
@@ -0,0 +1,390 @@
+
+##### Interact with Transaction Allow List
+
+Recommended file name: `ITxAllowList.sol`
+
+Precompile interface: [Transaction Allow List](./precompiles/AllowList.md)
+
+Precompile address: `0x0200000000000000000000000000000000000002`
+
+There are 2 main roles for Transaction Deployer Allow List precompile; `Admin` and `Allowed`.
+
+- `Admin`
+  - Can add new admins and alloweds
+  - Can send transactions
+- `Allowed`
+  - Can send transactions
+
+To check the role of an address run `readAllowList` function. It returns 0, 1 or 2, corresponding to the roles `None`, `Allowed`, and `Admin` respectively.
+
+##### Interact with Native Minter
+
+Recommended file name: `INativeMinter.sol`
+
+Precompile interface: [Native Minter](./precompiles//NativeMinter.md)
+
+Precompile address: `0x0200000000000000000000000000000000000001`
+
+There are 2 main roles for NativeMinter precompile; `Admin` and `Minter`.
+
+- `Admin`
+  - Can add new admins and minters
+  - Can mint native token
+- `Minter`
+  - Can mint native token
+
+To check the role of an address run `readAllowList` function. It returns 0, 1 or 2, corresponding to the roles `None`, `Minter`, and `Admin` respectively.
+
+## Customize the Subnet
+
+- `VM`: To understand and create your custom VM you can refer to [this](https://docs.avax.network/subnets/create-a-virtual-machine-vm).
+- `ChainId`: You want your `ChainId` parameter to be unique. To make sure that your Subnet is secure against replay attacks. To see registered `ChainIds` you can check [chainlist.org](https://chainlist.org/). At the top right of the site make sure to turn on the button to include testnets.
+- `Gas Parameters`: Ava Labs recommends the low-low option and C-Chain currently uses this option. But, if you know what you are doing you are free to customize. Note that higher disk usage has some trade offs, it would require more processing power and cause it to be more expensive to maintain.
+- `Airdrop Address`: You would not like to use the default address in production, that is recieving the 1 million tokens. Because, it is a compromised wallet, which means that its private key is well known by others. If you add a custom address to recieve airdrop. Avalanche-cli will ask you to give an amount in AVAX, in that case do not enter the value thinking as in `ether` but in `gwei` to correctly airdrop the amount you want. As an example, to airdrop `1` whole token, as in one ether, you would enter the value `1000000000`.
+- `Precompiles`: You can learn what precompiles are by refering to [this](https://docs.avax.network/subnets/customize-a-subnet#precompiles).
+
+## Troubleshoot Common Issues
+
+### Step 1: Install Avalanche-cli
+
+- `avalanche`, `avalanche subnet`
+  - `"command not found: avalanche"`
+    It means that `avalanche` command is not added to the PATH environment variable. It could be caused by following reasons;
```

Comment:
```suggestion
    It means that the directory containing `avalanche` command is not added to the PATH environment variable. It could be caused by following reasons;
```